### ISSUE TITLE
Side loading script-lab possible?

### ISSUE THREAD

**Issue by ComputerFields** [OfficeDev/script-lab#743]
Hello, I posted a question about side loading script-lab in stack overflow, but have not received a reply:

https://stackoverflow.com/q/56691383/11677488

Can anyone assist?

Thanks.

**Comment by Zlatkovsky** [comment]
It's a great question, but I'm afraid it's a general question/issue specific to Office Add-ins, not something specific to Script Lab.

It might well be that whatever admin settings are preventing "individual acquisition of Office Store Add-ins" are also preventing sideloading (which you'd be effectively using to circumvent the policy).

Can you try on another machine or a personal machine, and/or try in Office Online?

**Comment by ComputerFields** [comment]
I'm able to side-load and interact with other manifests locally; one that's serving up a local Visual Studio/IIS hosted Add-In, and another that just points to Google.com, so I know side-loading is not being blocked by policy or configuration.

However, I notice if any URL contained within the manifest isn't accessible, the Add-In doesn't show up in the Shared Folder list of Add-Ins.

Is anyone else able to side load the Script Lap manifest via a shared folder and get it to function as if it were downloaded from the Office Store?

**Comment by Zlatkovsky** [comment]
That's bizzare.  Let me re-open the issue.
Can you tweak the manifest to remove the troublesome URLs, and let me know what they are?
And just to be sure:  you are using the https://github.com/OfficeDev/script-lab/blob/master/manifests/script-lab-react-prod.xml one, right?

**Comment by ComputerFields** [comment]
Sorry, I should have been more clear. When I side-loaded the Visual Studio generated manifest, the handful of default URL's inside were formatted like this: "~remoteAppUrl/Home.html"

I had to fix them up like this so they pointed to the local IIS instance: "https://localhost:44354/Home.html"

After that, the Add-In would show up in the list of Shared Folder Add-Ins.

Script Lab has 65 URL's.... so I was hoping someone could tell me if what I'm attempting is even possible before I start testing each of them.

Yes, looks like I'm using the one you mention:
https://github.com/OfficeDev/script-lab/blob/master/manifests/script-lab-react-prod.xml

Thank you.

**Comment by ComputerFields** [comment]
Here are two manifests that work: 

[WordWebAddIn1.zip](https://github.com/OfficeDev/script-lab/files/3340878/WordWebAddIn1.zip)

**Comment by ComputerFields** [comment]
I think I've found the issues....

- I was right-clicking on the script-lab-react-prod.xml file and doing a "save as" to download the file from Github. That seems to modify the destination file.
- I attempted to view the file raw in Github and copy/paste into a new file, but I use Unix file types by default.

Now that I've overcome my self-inflicted problems, I'm running the Script Lab manifest side-loaded as-is and it seems to be functional.

Thank you.
